docs/system-admin-guide/files/external-file-storages/health-status/README.md

```diff
@@ -98,14 +96,14 @@ the [system admin guide](../../../../system-admin-guide/integrations/nextcloud/#
-| WRN_OIDC_NON_OIDC_USER            | The user is not a OIDC user.                               | The current user is provisioned to OpenProject, but not by an OIDC identity provider.                                                                                                                                 | Log in to an admin user with the correct OIDC identity provider, the same one used by the file storage, and rerun the check. |
-| WRN_OIDC_NON_PROVISIONED_USER     | The user is not a provisioned user.                        | The current user is not provisioned by an OIDC identity provider.                                                                                                                                                     | Log in to an admin user with the correct OIDC identity provider, the same one used by the file storage, and rerun the check. |
-| WRN_OFFLINE_ACCESS_SCOPE_MISSING  | The OIDC provider has no `offline_access` scope.           | The OIDC provider was configured without `offline_access` scope. While this does not cause the integration to break, this scope ensures, that refresh tokens to not expire and OpenProject can maintain a connection. | Add the scope `offline_access` to the OIDC provider configuration.                                                           |
+| Error code                        | Error description                                          | Possible reasons                                                                                                                                                                                                      | Next steps and solutions                                                                                                                                       |
+|-----------------------------------|------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ERR_OIDC_TOKEN_ACQUISITION_FAILED | No token could be acquired for accessing the file storage. | The OpenID Connect (OIDC) setup doesn't have the necessary audience, nor does it provide the possibility to exchange tokens.                                                                                          | Please check if you configured the correct file storage audience in OpenProject.                                                                               |
```

Comment:
I am wondering whether we should hint at the possibility of us not knowing about token exchange capability.

Though, I can't come up with a way to explain this, without making everything complicated and unreadable.

Reply:
We can extend this at any time, when we have better ideas or insight. I
'd rather not enforce it now.